styles.css

```diff
@@ -479,347 +447,307 @@ footer {
 .event-div p {
   color: white;
   margin-top: 20px;
-  
 }
 
 /* About-footer */
 
-#about-footer{
-  background-color: #272A31;
+#about-footer {
+  background-color: #272a31;
 }
```

Comment:
- [ ] [REQUIRED] Good job so far! It would be nice if you place the `footer texts` as like a [Behance Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) because your `paragraph` text stuck between border of section🙏🏼 

![image](https://github.com/Xghayor/Music-Festival/assets/63915024/51c82212-de7a-441b-9a9c-577bb1f2fa2d)